### PR TITLE
Issues #SB-5233 feat: changes for api-cleanup

### PR DESCRIPTION
**Pull request by manzarul**
Cleanup of un-used api

### CODE REVIEW

File: service/conf/routes [hunk 1]
```diff
@@ -2,11 +2,9 @@
 # This file defines all application routes (Higher priority routes first)
 OPTIONS    /*all                        @controllers.LearnerController.preflight(all)
 # ~~~~
-GET      /v1/user/courses/:uid	        @controllers.LearnerController.getEnrolledCourses(uid:String)
 GET      /v1/user/courses/list/:uid	        @controllers.LearnerController.getEnrolledCourses(uid:String)
 POST     /v1/user/courses/enroll	    @controllers.LearnerController.enrollCourse()
 POST     /v1/course/enrol	    		@controllers.LearnerController.enrollCourse()
-POST     /v1/user/content/state	        @controllers.LearnerController.getContentState()
```
Comment (bvinayakumar) [inline service/conf/routes]: Check if portal or mobile is using /v1/user/content/state.

File: service/conf/routes [hunk 2]
```diff
@@ -17,7 +15,6 @@ PATCH	/v1/user/update					@controllers.usermanagement.UserController.updateUserP
 POST	/v1/user/login					@controllers.usermanagement.UserController.login()
 POST	/v1/user/logout					@controllers.usermanagement.UserController.logout()
 POST	/v1/user/changepassword			@controllers.usermanagement.UserController.changePassword()
-GET  	/v1/user/getprofile/:uid			@controllers.usermanagement.UserController.getUserProfile(uid:String)
```
Comment (bvinayakumar) [inline service/conf/routes]: Check if portal or mobile is using /v1/user/content/state. Or it is onboarded or not.

File: service/conf/routes (hunk 1 above)
Comment (manzarul) [reply service/conf/routes]: onboarded  uri is /v1/content/state/read , actually inside routes /v1/user/content/state and /v1/content/state/read is pointing to same Controller method.

File: service/conf/routes (hunk 2 above)
Comment (manzarul) [reply service/conf/routes]: onboarded  uri is /v1/content/state/read , actually inside routes /v1/user/content/state and /v1/content/state/read is pointing to same Controller method.

**Pull request merged**